src/main/java/seedu/duke/commands/DeadlineCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.duke.commands;
+
+import seedu.duke.trackables.Deadline;
+import seedu.duke.trackables.Task;
+
+import java.util.List;
+
+public class DeadlineCommand extends Command {
+
+    private Deadline deadline;
+
+    public DeadlineCommand(Deadline deadline) {
+        this.deadline = deadline;
+    }
+
+    @Override
+    public void execute(List<Task> tasks) {
+        tasks.add(deadline);
+        echo(
```

Comment:
The indentation for wrapped lines in execute method should be 8 spaces.

Reply:
Line 20 should be together with Line 19.